src/Test.Utilities/CSharpCodeFixVerifier`2.cs

```diff
@@ -51,5 +51,20 @@ public static async Task VerifyCodeFixAsync(string source, DiagnosticResult[] ex
             test.ExpectedDiagnostics.AddRange(expected);
             await test.RunAsync();
         }
+
+        public static async Task VerifyAnalyzerAsync(
```

Comment:
📝 New helper methods should not be added to this class. I will remove this in a follow-up PR.